content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -133,6 +142,42 @@ This requires you have `openssl` installed on your machine.
     /tmp/pod-cert-chain-workload.pem: OK
     {{< /text >}}
 
+## (Optional) Generating root and intermediate keys and certificates offline
+
+This section gives you instructions on how to generate the keys and certificates offline and apply to
+the Istio cluster.
+
+On an offline machine with good security (restricted access), follow the `Create the root pair` and the
+`Create the intermediate pair` sections in this [guide](https://jamielinux.com/docs/openssl-certificate-authority/)
+to create the keys and certificates for your root and intermediate CA.
+
+After the certificates and keys are created,
+the root certificate is in `/root/ca/certs/ca.cert.pem`,
+the intermediate certificate is in `/root/ca/intermediate/certs/intermediate.cert.pem` and the intermediate key
+is in `/root/ca/intermediate/private/intermediate.key.pem`. We need to bundle them and copy them into the machine
+to manage the cluster.
+
+{{< text bash >}}
```

Comment:
since you are now providing steps in the form of bullets, this section will need to have 4 spaces inserted in the beginning - lines 160-167.